source/backend/api/Areas/Reports/Models/Management/ManagementActivityInvoicesReportModel.cs

```diff
@@ -169,5 +175,22 @@ private static string GetActivitySubTypesAsString(ICollection<PimsMgmtActivityAc
             }
             return string.Empty;
         }
+
+        private static string GetRegionsAsString(PimsManagementActivity activity)
+        {
+            if (activity is not null)
+            {
+                var activityRegions = activity.PimsManagementActivityProperties
+                        .Select(map => map?.Property?.RegionCodeNavigation?.Description)
+                        .Where(s => !string.IsNullOrWhiteSpace(s))
+                        .Distinct().ToList();
+                if (activity?.ManagementFile?.RegionCode != null)
+                {
+                    activityRegions.Add(activity?.ManagementFile?.RegionCodeNavigation?.Description);
+                }
```

Comment:
This section is correctly removing duplicate regions from multiple properties associated with the activity. But then it adds the management file region (without checking if that region was already in the list). This could lead to a duplicate value in the final list.

I would recommend using `.ToHashSet()` instead of `.Distinct().ToList()` so that when we try to add the file region it automatically discards potential duplicates. Thoughts?